scripts/predict.py

```diff
@@ -10,36 +11,48 @@
 
 @click.command()
 @click.option('--mask_path', required=True,
-              help='Adress of json file of the mask')
+              help='Path of json file of the mask')
 @click.option('--picture_path', required=True,
-              help='Adress of picture to be predicted')
+              help='Path of picture to be predicted')
 @click.option('--model_path', required=True,
```

Comment:
It seems to me this is mostly a question of style, not function. I do not have a particular stance here.